package Exceptions;

public class TestClass01 {

    public static void main( String [] args){

//        int i = 0;
//
//        test : {
//            System.out.println(" - - - - - - - - - - - - ");
//            try {
//                for ( ; true ; i++ ) {
//                    if( i > 5 ) break test;
//                }
//            } catch ( Exception e ) {
//                System.out.println(" in catch block ");
//            } finally {
//                System.out.println(" in finally block ");
//            }
//        }
    }

    public static void throwExc(){

//        throw new Exception();

    }
}
